### PR TITLE
show file path

### PR DESCRIPTION
**Pull request by gucong3000**
showfilename

### CODE REVIEW

**Comment by MoOx** [comment]
Nice!

File: index.js [hunk 1]
```diff
@@ -63,7 +63,13 @@ module.exports = postcss.plugin('postcss-browser-reporter', function (opts) {
 
         var content = warnings.map(function (message) {
             return warningToString(message).replace(/"/g, '\\"');
-        }).join('\\00000a');
+        });
```
Comment (MoOx) [inline index.js:66]: Good catch!

**Comment by gucong3000** [comment]
@MoOx  Will you merge it?

**Comment by MoOx** [comment]
I can but this repo is handled by @gazay (and I don't have any right on this package on npm so I won't be able to cut a release)...

**Comment by gazay** [comment]
Sorry for delay

**Comment by gazay** [comment]
v 0.5.0 published with this PR. Thank you for help @gucong3000 !

**Comment by gucong3000** [comment]
:+1:

**Pull request merged**